boards/same54-xpro/include/periph_conf.h

```diff
@@ -115,10 +115,6 @@ static const uart_conf_t uart_config[] = {
         .dev      = &SERCOM2->USART,
         .rx_pin   = GPIO_PIN(PB, 24),
         .tx_pin   = GPIO_PIN(PB, 25),
-#ifdef MODULE_PERIPH_UART_HW_FC
-        .rts_pin  = GPIO_UNDEF,
-        .cts_pin  = GPIO_UNDEF,
-#endif
```

Comment:
Why did you remove those one ?

Reply:
It's not needed, we check `.tx_pad == UART_PAD_TX_0_RTS_2_CTS_3`